lib/services/API/country_institute_api.dart

```diff
@@ -50,7 +50,7 @@ class HttpCountryInstituteAPI implements CountryInstituteAPI {
 
   @override
   Future<dynamic> getInstitutes(String query) async {
-    var url = 'http://universities.hipolabs.com/search?';
+    var url = 'http://universities.hipolabs.com/search?name=$query';
```

Comment:
Doesn't this duplicate the query, update _fetchAPI method if parameter format is changed